### PR TITLE
Allowing to use included views transformer from the current view

### PR DESCRIPTION
**Pull request by bhooshiek-narendiran**
:
Fix for [225](https://github.com/procore-oss/blueprinter/issues/225)

### CODE REVIEW

**Review by lessthanjacob — COMMENTED** [review]
Thanks for revisiting this! 

Since we're altering behavior here, could we some additional test assertions? You might be able to repurpose #247

File: lib/blueprinter/view_collection.rb [hunk 1]
```diff
@@ -35,7 +35,16 @@ def fields_for(view_name)
     end
 
     def transformers(view_name)
-      views[view_name].transformers
+      transformers = gather_transformers_from_included_views(view_name)
+      transformers.presence&.uniq || views[:default].transformers
```
Comment (njbbaer) [inline lib/blueprinter/view_collection.rb:39]: `presence` is a Rails specific method. How about this instead?
```suggestion
      transformers && !transformers.empty? ? transformers.uniq : views[:default].transformers
```

**Comment by njbbaer** [comment]
@bhooshiek-narendiran I'm hoping to move on this fix soon, so I've opened a new PR with some changes and tests. It's based on your commits, so they aren't lost. Please comment there if you have feedback, and let me know if you were planning any additional changes we can merge together.

**Comment by bhooshiek-narendiran** [comment]
> @bhooshiek-narendiran I'm hoping to move on this fix soon, so I've opened a new PR with some changes and tests. It's based on your commits, so they aren't lost. Please comment there if you have feedback, and let me know if you were planning any additional changes we can merge together.

Yes, we can merge this. I wrote the test. Facing some issues in local. Will try to complete it.

**Pull request closed without merging**